src/blockstack_cli.rs

```diff
@@ -148,6 +150,12 @@ The decode-tx command decodes a serialized Stacks transaction and prints it to s
 The transaction, if given, must be a hex string.  Alternatively, you may pass - instead, and the
 raw binary transaction will be read from stdin.";
 
+const DECODE_HEADER_USAGE: &str = "blockstack-cli (options) decode-header [block-path-or-stdin]
+
+The decode-header command decodes a serialized Stacks header and prints it to stdout as JSON.
+The header, if given, must be a hex string.  Alternatively, you may pass - instead, and the
```

Comment:
Back quoting `-` will make the last sentence more clear.

Reply:
Done